learning_assistant/utils.py

```diff
@@ -0,0 +1,42 @@
+"""
+Utils file for learning-assistant.
+"""
+import logging
+
+import requests
+from django.conf import settings
+from requests.exceptions import ConnectTimeout
+from rest_framework import status as http_status
+
+log = logging.getLogger(__name__)
+
+
+def get_chat_response(message_list):
+    """
+    Pass message list to chat endpoint, as defined by the CHAT_COMPLETION_API setting.
+    """
+    completion_endpoint = getattr(settings, 'CHAT_COMPLETION_API', None)
+    if completion_endpoint:
+        headers = {'Content-Type': 'application/json'}
+        connect_timeout = getattr(settings, 'CHAT_COMPLETION_API_CONNECT_TIMEOUT', 1)
+        read_timeout = getattr(settings, 'CHAT_COMPLETION_API_READ_TIMEOUT', 10)
+
+        try:
+            response = requests.post(
+                completion_endpoint,
+                headers=headers,
+                data=message_list,
+                timeout=(connect_timeout, read_timeout)
+            )
+            chat = response.json()
+            response_status = response.status_code
+        except (ConnectTimeout, ConnectionError) as e:
+            error_message = str(e)
+            log.error('Failed to connect to chat completion API. %(error)s', {'error': error_message})
+            chat = error_message
```

Comment:
Do we want to send back a Python error as the chat response in this case?

I imagine we do not want that displayed in the UI or visible in the network tab. What's our plan for handling this case in the UI?